<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectExplorer::Internal::ProcessStepWidget</class>
 <widget class="QWidget" name="ProjectExplorer::Internal::ProcessStepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="enabledGroupBox">
     <property name="title">
      <string>Enable custom process step</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="commandLabel">
        <property name="text">
         <string>Command:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Core::Utils::PathChooser" name="command"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="workingDirecoryLabel">
        <property name="text">
         <string>Working Directory:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Core::Utils::PathChooser" name="workingDirectory"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="commandArgumentsLabel">
        <property name="text">
         <string>Command Arguments:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="commandArgumentsLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>80</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Core::Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
